### PR TITLE
created Rules page

### PR DESCRIPTION
**Pull request by Um-Brehon**

### CODE REVIEW

File: src/Components/Rules/index.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
+@media only screen and (min-width: 700px) {
+  .rulesBody {
+    color: var(--secondary-color);
+    width: 90%;
+
+    /* width: 100%; */
```
Comment (Franck59840) [inline src/Components/Rules/index.css]: Why comment ?

**Review by Franck59840 — CHANGES_REQUESTED** [review]

**Review by Franck59840 — DISMISSED** [review]
good

**Review by LoicBrassart — DISMISSED** [review]
Saw no Evil

**Review by Vjerem — APPROVED** [review]
it's great for me

**Review by LoicBrassart — APPROVED** [review]
Saw no Evil

**Pull request merged**